src/createServer.js

```diff
@@ -1,3 +1,60 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    const errors = [];
+    const errorMessage = {
+      noOriginalText: 'Text to convert is required. '
+     + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      noTargetCase: '"toCase" query param is required. '
+     + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      unsupportedCase: 'This case is not supported. '
+     + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    };
+
+    const originalText = req.url.split('?')[0].split('/')[1];
```

Comment:
you can parse url using constructor [URL](https://nodejs.org/api/url.html#constructing-a-url-from-component-parts-and-getting-the-constructed-string). And use pathname from parsed url